app/policies/task_policy.rb

```diff
@@ -0,0 +1,31 @@
+class TaskPolicy < ApplicationPolicy
+  authorize :user, allow_nil: true
+
+  def index?
+    show?
```

Comment:
This is wrong. You can use one method for all, but it shouldn't be method for authorizing action like `show?`

Reply:
@RafaelKhammadeev